docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -25,9 +25,8 @@ yarn add aws-amplify @aws-amplify/ui-angular@3.x
 
 The following breaking changes were introduced in `@aws-amplify/ui-angular` major version `3.0`:
 
-#### Automatic signin on signup logic moved to `aws-amplify`. 
 
-##### 1. Update the handleSignUp function override call
+#### 1. Update the handleSignUp function override call
```

Comment:
We were using flat numbered list in React migration guide, so adjusting Angular guide formatting as so.